llvm/test/CodeGen/AArch64/partial-reduce-dot-product.ll

```diff
@@ -0,0 +1,96 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --version 4
+; RUN: opt -passes=loop-vectorize -force-vector-interleave=1 -S < %s | FileCheck %s
```

Comment:
tests for LV should be in llvm/test/Transforms/LoopVectorize

Also, does this need negative tests?